layouts/partials/page.html

```diff
@@ -4,6 +4,16 @@
   </div>
   <div class="page__content">
     <h1 class="page__title">{{ .Title }}</h1>
+    {{ if eq .Type "has-toc" }}
+    <div class="page__toc">
+      <h3 class="page__toc__title">On this page</h3>
+    {{ .TableOfContents }}
+    </div>
+    {{ end }}
+
+    {{ if eq .Type "release-note" }}
+    {{ end }}
```

Comment:
Do we want to keep this empty `release-note` block? I assume it's just WIP but wanted to confirm.

Reply:
Woops!  Deleted and force pushed.